cmd/backup_create.go

```diff
@@ -0,0 +1,261 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/AlecAivazis/survey/v2"
+	pb "github.com/odpf/optimus/api/proto/odpf/optimus/core/v1beta1"
+	"github.com/odpf/optimus/config"
+	"github.com/odpf/optimus/models"
+	"github.com/odpf/salt/log"
+	"github.com/pkg/errors"
+	cli "github.com/spf13/cobra"
+	"google.golang.org/grpc"
+)
+
+func backupCreateCommand(l log.Logger, datastoreRepo models.DatastoreRepo, conf config.Provider) *cli.Command {
+	var (
+		backupCmd = &cli.Command{
+			Use:     "create",
+			Short:   "Create a backup",
+			Example: "optimus backup create --resource <sample_resource_name>",
```

Comment:
in backup users need to put the resource name in a flag, while for replay it is an argument:
- optimus backup create --resource <resource_name>
- optimus replay create <job_name>
should we also make this part consistent?